### PR TITLE
Windows Support (Experimental)

### PR DESCRIPTION
**Pull request by johnzja**
Training & Testing supported under Windows.
READMEs updated.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -25,9 +25,26 @@ conda install pytorch=1.5.0 torchvision cudatoolkit=10.1 -c pytorch
 ```
 
 c. Install [MMCV](https://mmcv.readthedocs.io/en/latest/) following the [official instructions](https://mmcv.readthedocs.io/en/latest/#installation).
-Either `mmcv` or `mmcv-full` is compatible with MMSegmentation, but for methods like CCNet and PSANet, CUDA ops in `mmcv-full` is required
+Either `mmcv` or `mmcv-full` is compatible with MMSegmentation, but for methods like CCNet and PSANet, CUDA ops in `mmcv-full` is required.
```
Comment (xvjiarui) [inline docs/install.md:28]: We may separate Windows and Linux. 
And put Linux ahead.

Comment (johnzja) [reply docs/install.md:28]: OK.

**Comment by johnzja** [comment]
I'll use "merge" instead of "rebase" when i need to keep my branch updated with open-mmlab/mmsegmentation.

File: docs/getting_started.md [hunk 2]
```diff
@@ -221,6 +221,13 @@ evaluation = dict(interval=4000)  # This evaluate the model per 4000 iterations.
 **\*Important\***: The default learning rate in config files is for 4 GPUs and 2 img/gpu (batch size = 4x2 = 8).
 Equivalently, you may also use 8 GPUs and 1 imgs/gpu since all models using cross-GPU SyncBN.
 
+If you train a model on Windows, please add the following line to your config because symlink is not supported
+in os package under Windows.
+
+```python
+checkpoint_config = dict(create_symlink=False)
```
[the remark this answers is not in the document]
Comment (johnzja) [reply docs/getting_started.md]: fixed

File: docs/install.md [hunk 3]
```diff
@@ -17,22 +15,51 @@ conda activate open-mmlab
...
+
+Install mmcv for Linux:
 
 The pre-build mmcv-full (with PyTorch 1.5 and CUDA 10.1) can be installed by running: (other available versions could be found [here](https://mmcv.readthedocs.io/en/latest/#install-with-pip))
 
 ```shell
 pip install mmcv-full==latest+torch1.5.0+cu101 -f https://openmmlab.oss-accelerate.aliyuncs.com/mmcv/dist/index.html
 ```
 
+Install mmcv for Windows:
```
Comment (xvjiarui) [inline docs/install.md:36]:
```suggestion
**Install mmcv for Windows (Experimental):**
```

File: docs/install.md [hunk 4]
```diff
@@ -17,22 +15,51 @@ conda activate open-mmlab
...
 ```shell
-conda install pytorch=1.5.0 torchvision cudatoolkit=10.1 -c pytorch
+conda install pytorch=1.6.0 torchvision cudatoolkit=10.1 -c pytorch
 ```
 
 c. Install [MMCV](https://mmcv.readthedocs.io/en/latest/) following the [official instructions](https://mmcv.readthedocs.io/en/latest/#installation).
-Either `mmcv` or `mmcv-full` is compatible with MMSegmentation, but for methods like CCNet and PSANet, CUDA ops in `mmcv-full` is required
+Either `mmcv` or `mmcv-full` is compatible with MMSegmentation, but for methods like CCNet and PSANet, CUDA ops in `mmcv-full` is required.
+
+Install mmcv for Linux:
```
Comment (xvjiarui) [inline docs/install.md:28]:
```suggestion
**Install mmcv for Linux:**
```

**Review by xvjiarui — APPROVED** [review]

**Comment by sshuair** [comment]
@johnzja Have you test it on multi-gpu with DDP training?
